Helper/Webhook.php

```diff
@@ -109,20 +109,7 @@ public function __construct(
      */
     public function processNotification(Notification $notification): bool
     {
-        // set notification processing to true
-        $this->updateNotification($notification, true, false);
-        $this->logger
-            ->addAdyenNotification(
-                sprintf(
-                    "Processing %s notification %s",
-                    $notification->getEventCode(),
-                    $notification->getEntityId(),
-                ), [
-                    'merchantReference' => $notification->getMerchantReference(),
-                    'pspReference' => $notification->getPspreference()
-                ],
-            );
-
+        // check if merchant reference is set
         if (is_null($notification->getMerchantReference())) {
             $errorMessage = sprintf(
```

Comment:
I don't think this case should be marked as error, as we will just skip this notification from processing and log it

```suggestion
            $warningMessage = sprintf(
```